hs_core/tests/api/native/test_bag_ingestion_command.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+from unittest import TestCase
+
+from django.contrib.auth.models import Group
+from django.core.exceptions import ValidationError
+
+from hs_composite_resource.models import CompositeResource
+from hs_core.testing import MockIRODSTestCaseMixin
+from hs_core import hydroshare
+from django.core.management import call_command
+
+
+class TestIngestBag(MockIRODSTestCaseMixin, TestCase):
+
+    def setUp(self):
+        super(TestIngestBag, self).setUp()
+
+        self.hs_group, _ = Group.objects.get_or_create(name='Hydroshare Author')
+        # create a user
+        self.user = hydroshare.create_account(
+            'test_bag_ingestion@email.com',
+            username='bag_ingestion_test',
+            first_name='some_first_name',
+            last_name='some_last_name',
+            superuser=False,
+            groups=[self.hs_group],
+        )
+
+        # delete all resources in case a test isn't cleaning up after itself
+        CompositeResource.objects.all().delete()
+
+    def tearDown(self):
+        super(TestIngestBag, self).tearDown()
+        self.user.delete()
+        self.hs_group.delete()
+        CompositeResource.objects.all().delete()
```

Comment:
I am pretty sure Django automatically clean up objects in database in tearDown(), so no need to delete user and hs_group explicitly in tearDown() although there might be a need to delete all CompositeResource objects since I am not sure whether the associated files in iRODS containers would be deleted automatically by Django's automatic cleanup.

Reply:
I originally left those out but the tests failed when I ran all the tests.  I don't want to hunt down what's going on, I'd rather just ensure the environment is clean enough for these current tests.